### ISSUE TITLE
Not possible to save `commtype`

### ISSUE THREAD

**Issue by magfro** [hassio-addons/addon-appdaemon2#4]
# Problem/Motivation

After 0.2.0, commtype is required in options, but isn't there by default, making the following error appear in the logs when appdaemon is started:

```
usage: appdaemon [-h] [-c CONFIG] [-p PIDFILE] [-t TICK] [-s STARTTIME]
                 [-e ENDTIME] [-i INTERVAL]
                 [-D {DEBUG,INFO,WARNING,ERROR,CRITICAL}] [-v]
                 [--commtype {SSE,WEBSOCKETS}] [--convertcfg] [-d]
appdaemon: error: argument --commtype: invalid choice: '' (choose from 'SSE', 'WEBSOCKETS')

```

But commtype can't be added to options:
```
{
  "log_level": "info",
  "commtype": "websockets"
}

```

Gives the following error in supervisor logs:

`17-11-01 07:00:08 WARNING (MainThread) [hassio.addons.validate] Unknown options commtype
`

## Expected behavior

1. Have "commtype": "websockets" inserted as default in options.
2. Make it possible to change this value.

## Actual behavior

Error messages and unable to start appdeamon since last change.

## Steps to reproduce

1. Install appdaemon from repository.
2. Start it.
3. See logs.

## Proposed changes

See expected behavior above.

**Comment by frenck** [comment]
Thank you for this report. I've found the issue and will release a patch asap.

**Comment by frenck** [comment]
Fixed in v0.2.1.

Thanks again for the report!
